src/script/server/server.js

```diff
@@ -1,18 +1,24 @@
+const path = require('path')
 const express = require('express')
 const log = require('loglevel')
-const { PORT, PUBLIC_FOLDER } = require('./constants')
 const bundle = require('./bundle')
 const api = require('./api')
 
-const start = (component, webpackConfig) => {
+// FIXME: move it to root ?
```

Comment:
Yes, I think we can put `PORT` and `PUBLIC_FOLDER` in the state
And `PUBLIC_FOLDER=path.resolve(state.workbenchPath, 'public')`